docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,291 @@
+                                                             |                                                                   
+    +---------------------------------------+                |                                                                   
+    |Comment `Duplicate of #<issue number>` | YES +----------|----------+                                                        
+    |Remove needs-triage label              -------  Duplicate issue?   |                                                        
+    |label: triage/duplicate                |     |                     |                                                        
+    +-----|---------------------------------+     +-----------|---------+                                                        
+          |                                                   |NO                                                                
+          |  +-------------------------+  NO   +-----------------------------+                                                   
+          |  |Add comment              |--------  Is this a valid feature?   |                                                   
+          |  |Remove needs-triage label|       |                             |                                                   
+          |  +------|------------------+       +--------------|--------------+                                                   
+          |         |                                         | YES                                                              
+          |         |                                         |                                                                  
+          |         |                         +---------------|--------------+                                                                                                    
+          |         |                         | label: type/feature          |                                                   
+        +-|---------|---+       +--------+    | Remove needs-triage label    |                                                   
+        |  Close issue  |       |  Done  ------ Remove type/feature-request  |                                                   
+        |               |       |        |    | milestone?                   |                                                   
+        +---------------+       +--------+    +------------------------------+           
+```
+If the author does not respond to a request for more information within the timespan of a week, close the issue with a kind note stating that the author can request for the issue to be reopened when the necessary information is provided.
+
+In some cases you may receive a request you do not wish to accept.  Perhaps the request doesn't align with the project scope or vision.  It is important to tactfully handle contributions that don't meet the project standards.
```

Comment:
Please also fix the other instances where the word "contributions"  appears instead of "request".  There's one in this line and also instances on lines 286 and 288.